staging/src/kubevirt.io/api/migrations/v1alpha1/types.go

```diff
@@ -139,17 +139,17 @@ func (score migrationPolicyMatchScore) lessThan(otherScore migrationPolicyMatchS
 // deterministic way of matching policies.
 func (list *MigrationPolicyList) MatchPolicy(vmi *k6tv1.VirtualMachineInstance, vmiNamespace *k8sv1.Namespace) *MigrationPolicy {
 	var mathingPolicies []MigrationPolicy
-	mostMatchingLabels := 0
+	bestScore := migrationPolicyMatchScore{}
 
 	for _, policy := range list.Items {
-		doesMatch, curMatchingLabels := countMatchingLabels(&policy, vmi.Labels, vmiNamespace.Labels)
+		doesMatch, curScore := countMatchingLabels(&policy, vmi.Labels, vmiNamespace.Labels)
 
-		if !doesMatch || curMatchingLabels < mostMatchingLabels {
+		if !doesMatch || curScore.lessThan(bestScore) {
 			continue
-		} else if curMatchingLabels > mostMatchingLabels {
-			mostMatchingLabels = curMatchingLabels
+		} else if curScore.greaterThan(bestScore) {
+			bestScore = curScore
 			mathingPolicies = []MigrationPolicy{policy}
-		} else if curMatchingLabels == mostMatchingLabels {
+		} else if curScore.equals(bestScore) {
```

Comment:
Just to know: Is this `else if` explicitly written to increase readability? Thanks

Reply:
Yes.
Before this PR this code was much less readable, so I thought it's needed. But after this PR I guess the code it very straight forward there, so I guess it can be removed. Thanks again!

Reply:
I was just curious if I there were another reason that I cannot figure out. Thanks :)